sdk/python/tests/unit/permissions/test_oidc_auth_client.py

```diff
@@ -0,0 +1,62 @@
+from unittest.mock import patch
```

Comment:
Didn't Daniele write a test with mocks for oidc, does this test adds additional validations or flows to his test?

Reply:
this is unit testing. Daniel's is having integration testing. There will be some overlap.